ffi/framebuffer_android.lua

```diff
@@ -149,34 +176,39 @@ end
 function framebuffer:refreshPartialImp(x, y, w, h)
     self:_updateWindow()
     if has_eink_full_support then
-        self:_updatePartial(partial_regal, 0, x, y, w, h)
+        -- in case of qualcomm it's actually partial_gc16
```

Comment:
Maybe makes sense to use a name that represents our intent instead of the waveform and mode. That would mean calling it "full", "partial", "ui", "partialUi" and "fast" instead of "full_gc16", "partial_regal", "full_regal", "partial_du".